### ISSUE TITLE
Why scale down SDD by 100?

### ISSUE THREAD

**Issue by centiLinda** [cocoon2wong/Vertical#8]
Hi author, thanks for sharing your work!

I noticed that you chose to scale down the SDD raw trajectories by 100 during training. What is the reason for this?

I'm having trouble running my own model on SDD. The predicted distribution seems random, and the most-likely trajectory tends to be a static point. However, I do not have this problem with ETH-UCY dataset.

The major difference between these 2 datasets (that I can think of for now) is the scale of the data, one in pixel and one in meters. So I wonder maybe you faced the same issue when training SDD and that's why you chose to scale it down?

**Comment by cocoon2wong** [comment]
Hi @centiLinda !
Thanks for your question!

Considering that most of the SDD scenes are 1920*1080 videos (although there may be small differences) and that the scale in the ETH-UCY scenes (e.g., from the leftmost to the rightmost side of the video) is roughly 10 to 20 meters, we used this parameter to make the two datasets have roughly the same scale. However, if trained and tested only on SDD, we think that this one constant scaling of 100 should not affect the model.

As for the problem you mentioned with the model's predictions becoming a STATIC POINT, while I'm not sure if this is related to the structure of your model, I think that a “moving” preprocessing of the trajectory might be a solution, i.e., making the coordinates of a moment in the trajectory (0, 0) before training, and after the model prediction moving them back, like what we have done when training our models:

https://github.com/cocoon2wong/Vertical/blob/178866cf547150dc98d18817713e257aff7429f9/codes/basemodels/__preprocess.py#L16-L29

I hope this helps!

**Comment by centiLinda** [comment]
Thanks for your quick reply!

I just ran a quick experiment where I scaled down the trajectories by 100, and it worked! For now I have no idea why this works, I'll take a closer look and update here if I discover anything.

If I understand correctly, your suggested preprocessing is somewhat like a normalization, to make all samples start from (0, 0). It's probably not related to my issue, but thanks for pointing it out!

All the best!
